config.lib

```diff
@@ -1202,6 +1202,12 @@ check_params() {
 	fi
 }
 
+if ! which realpath; then
+	realpath() {
+		[[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
```

Comment:
I'm pretty sure there's nothing bash-specific in config(.lib|ure) so far, do you think you can make it bourne-shell compliant?

Reply:
Sure. Replacing with one that worked on busybox's `sh`.

Reply:
Please let me leave it the way it is for now. I don't feel I have the resources to make it portable to DOS right now.

Reply:
Please don't worry about DOS and mingw for now :) They are already on their own path (pun intended). Happy bashism is gone :)

Reply:
OK, `if` deleted.